tests/bgp/test_bgp_authentication.py

```diff
@@ -255,7 +255,7 @@ def test_bgp_neighbor_password(setup):
 
     # remove password configs
     cmd = 'vtysh -n {} -c "config" -c "router bgp {}" -c "no neighbor {} password {}" -c \
-        "no neighbor {} password {}" -c "end"'.format(setup['neigh_asic_index'], setup['dut_asn'],
+        "no neighbor {} password {}" -c "end"'.format(setup['asic_index'], setup['dut_asn'],
```

Comment:
can you add check sth like:
ns = "-n " + setup['asic_index'] if asic_index, else ""
cmd = 'vtysh '+ ns + ' -c ....'